### ISSUE TITLE
3 way binding and live saving = strange behaviour

### ISSUE THREAD

**Issue by Xample** [Urigo/angular-meteor#346]
I am binding a value on an <input> tag, my object is in live save mode. The problem I got is that everytime I press a key, the model is changed, sent to the server, then the server send me back the version and replace my current model one. How could this be solved ? I thought about throttling the event but it will still be the same case, to only save on blur() might be an option to consider, disabling live saving - reenabling could also be another option but how to achieve so ? Any suggestion is welcome

**Comment by auterium** [comment]
Did you check for update permission? If you remove insecure package, you must add explicit update permission to the collection, otherwise it will reject any change.

**Comment by Urigo** [comment]
yes that sound like a reasonable option..

**Comment by Xample** [comment]
@auterium sure I did, it's not a problem of the content not to be changed. It's a problem of the time to propagate to the server and the server sending back the changes. While disabling autosaving there is no problem because all is sent at once.
Normally with bindings we do know who initially changed the data to stop propagating at some point.
The best solution being:
- to keep a trace client side of the data sent to the server
- when the server send those data back to the client:
- if the data are the same, we dismiss those changes i.e. we do not update the client
- otherwise we update the client with the server's data.

A limitation to this approach is while having several changes per second and therefore caching a lot of data. This is where throttling might be an option. i.e. we send the data at a maximum rate of 1 time per second.

**Comment by Urigo** [comment]
@Xample can you please share your code?
Is it possible that it sends just one update and then loses focus?
if so, it a problem I am working on right now

**Comment by Urigo** [comment]
@Xample have you tried with the latest version?

**Comment by Urigo** [comment]
@Xample I'm closing as I'm still waiting for update.
Please feel free to re-open or email me if the problem persists.

**Comment by Xample** [comment]
@Urigo Hi, sorry I was on holidays and I dropped my computer to the applecare in the same time, I got it back this morning. I solved the problem using ```ng-model-options="{ updateOn: 'blur' }"``` with live edition, the problem was still there. I will start another project very soon also with the same edition features, I will try to periodically check if this bug occurs, if so I will then have a smaller project able to reproduce the bug. I also had the blurring bug you described (every key made the editor loose focus) and also solved it adding the updateOn blur option. I updated right after your update.
